specification/managementpartner/resource-manager/Microsoft.ManagementPartner/preview/2018-02-01/examples/GetPartnerDetailsNoPartnerId.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "api-version": "2018-02-01"
```

Comment:
@jetts you updated existing spec with new tags.
Do you mind adding an example that shows the request that has the newly added tags.